### DISCUSSION TITLE
UWB cannot play embedded video in html5

### DISCUSSION THREAD

**Discussion by longkakavn** [Voltstro-Studios/UnityWebBrowser#245] [Q&A]
I'm using UWB to play html5 file with embedded video (video in local folder) but it doesn't work. It shows error "[Web Browser]: [CEF Engine Console]: Uncaught (in promise) NotSupportedError: The element has no supported sources.". I tried different ways: open html5 as file, open via localhost/remote host but it didn't work. It only worked with embedded video from youtube.
Anyone knows how to fix this, please?
Thanks in advanced!

**Comment by Voltstro** [comment]
Video cannot be any proprietary codecs. See https://github.com/Voltstro-Studios/UnityWebBrowser/discussions/79#discussioncomment-2758713
